src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -357,42 +357,42 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             ' grab the part that doesn't contain the preceding and trailing trivia.
 
             Dim builder = PooledStringBuilder.GetInstance()
-            Dim writer As New IO.StringWriter(builder)
+            Using writer As New IO.StringWriter(builder)
```

Comment:
>Using [](start = 12, length = 5)

From [https://docs.microsoft.com/en-us/dotnet/api/system.io.stringwriter?view=netframework-4.8](https://docs.microsoft.com/en-us/dotnet/api/system.io.stringwriter?view=netframework-4.8)

Note

This type implements the IDisposable interface, but does not actually have any resources to dispose. This means that disposing it by directly calling Dispose() or by using a language construct such as using (in C#) or Using (in Visual Basic) **is not necessary**.